cookbook/security/entity_provider.rst

```diff
@@ -458,30 +456,13 @@ interface requires three methods: ``loadUserByUsername($username)``,
 
             return $user;
         }
-
-        public function refreshUser(UserInterface $user)
-        {
-            $class = get_class($user);
-            if (!$this->supportsClass($class)) {
-                throw new UnsupportedUserException(
-                    sprintf(
-                        'Instances of "%s" are not supported.',
-                        $class
-                    )
-                );
-            }
-
-            return $this->find($user->getId());
-        }
-
-        public function supportsClass($class)
-        {
-            return $this->getEntityName() === $class
-                || is_subclass_of($class, $this->getEntityName());
-        }
     }
 
-For more details on these methods, see :class:`Symfony\\Component\\Security\\Core\\User\\UserProviderInterface`.
+.. versionadded:: 2.8
+    The :class:`Symfony\\Bridge\\Doctrine\\Security\\User\\UserLoaderInterface`
+    interface was introduced in 2.8. Prior to Symfony 2.8, you had to implement
+    `Symfony\\Component\\Security\\Core\User\\UserProviderInterface` requiring you
```

Comment:
you should use double backticks in reStructured Text and only a single `\`. I would also prefer to stop after the class name, removing "requiring you to implement three methods, where only one was useful.".